### ISSUE TITLE
Generating random outputs after some epochs of Training

### ISSUE THREAD

**Issue by jackwardphines** [junyanz/pytorch-CycleGAN-and-pix2pix#680]
I am training a CycleGAN on my own data set of 10000 tables. Input is a natural table and output should be the skeleton of the table. For the skeleton I'm generating random table skeletons using a python script. The training went OK through first 5-8 epochs and after that it is generating some random tables with random noise. Also the GAN_A loss is oscillating so much and my batch size is 4. Does cycle GAN become worse after few epochs of training or Am I doing something wrong?. Attached the loss function with respect to the number of epochs(25). 
![newplot](https://user-images.githubusercontent.com/50595050/59738650-9cc26c00-927f-11e9-9d7c-0c332508fa58.png)

**Comment by junyanz** [comment]
It's hard to tell. Could you try batch_size=1? Also, not sure if you can transfer a table into a randomly generated skeleton. The quality of the translation will depend on how you generate random skeletons.

**Comment by Vaishnvi** [comment]
Hi @junyanz ,

I have been training cyclegan on my own dataset of around 500 images of A and 500 images of B. 
But after few epochs I see that there’s random noise added to the fake images being generated while training and after 60 epochs now it’s still having some noise in generated images.
But I see that g_loss is minimising well for 60 epochs as of now. 
Is there anything that’s going wrong in training ? 

![296AA292-470A-44F8-9D5D-C277E280E3B1](https://user-images.githubusercontent.com/24861262/84157901-e146a900-aa88-11ea-8484-5310e629eb76.jpeg)

**Comment by junyanz** [comment]
The training looks normal to me. The noise might go away if the model is trained longer. If not, you can also look at #64

**Comment by junyanz** [comment]
Feel free to change the number of epochs. I think it depends on your datasets and applications, and hard to tell in advance.

**Comment by Vaishnvi** [comment]
Yeah will check. Thanks a lot for help :) 
Also your work and repo is really very helpful !
Thanks :)
